frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -10,18 +10,34 @@ type SupportedAppTitleProps = {
 };
 
 const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvider = false }) => {
+  const { title, icon } = React.useMemo(() => {
+    const split = odhApp.spec.displayName?.split(' ');
+
+    if (isRedHatSupported(odhApp)) {
+      return {
+        title: `${split.slice(0, -1).join(' ')} `,
```

Comment:
It's interesting -- if you have something like `Jupyter` -- you end up with no title. Not sure that really matters for display purposes -- but it might for screen readers.

Maybe we don't split the title when there is only 1 word?

Reply:
yeah I thought about that. I ended up doing this because it was less code. For screen readers, it will always be wonky if there is an icon, because the last word wont be in the title. So to mitigate the screen reader issue we should just use aria-labels for the icon and title.
title aria-labelledby: _full title text_
icon aria-label: "Application is supported by Red Hat"

Reply:
Good point! Sounds like a plan to me.